signer/tests/integration/transaction_coordinator.rs

```diff
@@ -1167,8 +1174,13 @@ async fn sign_bitcoin_transaction() {
 
     let chain_tip_info = rpc.get_chain_tips().unwrap().pop().unwrap();
 
-    // 1. Create a database, an associated context, and a Keypair for each of
-    //    the signers in the signing set.
+    // =========================================================================
```

Comment:
Nice comment style :+1: :smile:

Reply:
yeah, 😂 I like them a lot, thank you for introducing them here. Now all I need some extension to make adding them easier.